src/pages/ManageVaccineStockPage.js

```diff
@@ -55,33 +55,33 @@ const TABLE_COLUMNS = {
   code: { key: 'code', width: 0.6, title: LOCALIZATION.headers.code, alignText: 'left' },
   totalBatchesInStock: {
     key: 'totalBatchesInStock',
-    width: 0.8,
+    width: 0.5,
     title: LOCALIZATION.headers.totalBatchesInStock,
-    alignText: 'right',
+    alignText: 'center',
   },
   totalQuantity: {
     key: 'totalQuantity',
     width: 0.5,
     title: LOCALIZATION.headers.totalQuantity,
-    alignText: 'right',
+    alignText: 'center',
   },
   hasBreach: {
     key: 'hasBreach',
     width: 0.5,
     title: LOCALIZATION.headers.hasBreach,
-    alignText: 'left',
+    alignText: 'lecenterft',
```

Comment:
is that some sort of yoga alignment hehe

Reply:
78f258f5b45abe12a5526d2aa9e1c3f17daa428b